README.md

```diff
@@ -346,11 +346,11 @@ Hurray! You successfully made a contribution!
 
 - ### **R**
 
-  - [Rakesh] (https://github.com/RakeshPotnuru)
   - [Rahul](https://github.com/kohli6010)
   - [Raj Vaya](https://github.com/rajvaya)
   - [Raju Ghorai](https://github.com/coderj001)
   - [Rahman Haroon](https://github.com/rahmanharoon)
+  - [Rakesh] (https://github.com/RakeshPotnuru)
```

Comment:
Please remove the extra space after the ending array bracket, then it should be ok to merge